fetch-crypto-assets.js

```diff
@@ -0,0 +1,65 @@
+const file = "./data.json"
+
+async function getJSON(jsonFile) {
+    try {
+        const response = await fetch(jsonFile)
+        const data = await response.json()
+        return data.cryptoAssets
+    } catch (e) {
+        throw new Error("Retrieving JSON failed", e)
+    }
+}
+
+function buildURL(param) {
+    return `https://api.coinbase.com/v2/prices/${param}/spot`
+}
+
+async function getURLS() {
+    try {
+        const params = await getJSON(file)
+        const urls = params.map((val) => {
```

Comment:
you can simply return immediately

```js
return params.map(...)

Reply:
Addressed: fc516e8a0fc15b17133c902a76d79e362313af8e